client/src/users/Users.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+class Users extends Component {
+  state = {
+    users: [],
+    status: null
+  };
+
+  componentDidMount() {
+    const token = localStorage.getItem("token");
+    if (token) {
+      const headers = { authorization: token };
+      axios
+        .get(`http://localhost:2525/api/users`, { headers })
+        .then(res => this.setState({ users: res.data, status: res.status }))
```

Comment:
You could potentially make this slightly shorter/cleaner with object destructuring:
```javascript
  .then(({ data, status }) => this.setState({ users: data, status }))
```